docs/src/pages/components/skeleton.mdx

```diff
@@ -0,0 +1,183 @@
+`display` prop manually to get it to look just the way you want.
+
+---
+
+### Wrapping
+
+The second way to use the `Skeleton` is by wrapping an element that you want to cover. For instance,
+if you had a fixed size `img` that was still loading, you could wrap it with a Skeleton and it'd
+simply take the shape of your image and hide it.
+
+```tsx live
+<Component initialState={{ loading: true }}>
+    {({state, setState}) =>
+        <section>
+            <div style={{paddingBottom: '1rem'}}>
+                <Skeleton loading={state.loading}>
+                    <img src={withPrefix('/images/anchor-01.jpg')} />
+                </Skeleton>
+            </div>
+
+            <Button onClick={() => setState({loading: !state.loading})}>
+                Toggle Skeleton
+            </Button>
+        </section>
+    }
+</Component>
+```
+
+If the wrapped child has a `border-radius`, you'll probably want to give the the `Skeleton` the same
+`border-radius` (especially if the child is circular, like
+an <Link to="/components/avatar">`Avatar`</Link>).
+
+The `Skeleton` does not know the `display` setting of the child it's wrapping, so you may need to
+set it manually (i.e. if the child is a `block` you'll probably want to set the `Skeleton` to a
+`block` as well, so that it fits in the DOM the same way).
+
+---
+
+### Custom Shape
+
+You might just want to give the Skeleton specific dimensions to make it take the exact shape you desire.
+
+```tsx live
+<section style={{whiteSpace: 'normal'}}>
+    <Skeleton width="100%" height="0.5rem" borderRadius="circular" marginBottom="1rem" />
+    <Skeleton width="100%" height="0.5rem" borderRadius="circular" marginBottom="1rem" />
+    <Skeleton width="80%" height="0.5rem" borderRadius="circular" marginBottom="1rem" />
+</section>
+```
+
+<br />
+
+---
+
+## API
+
+<ApiTable data={[
+    {
+        property: 'width',
+        description: `Sets the CSS <pre>width</pre> property.`,
+        type: 'string',
+    },
+    {
+        property: 'height',
+        description: `Sets the CSS <pre>height</pre> property.`,
+        type: 'string',
+    },
+    {
+        property: 'maxWidth',
+        description: `Sets the CSS <pre>max-width</pre> property.`,
+        type: 'string',
+    },
+    {
+        property: 'maxHeight',
+        description: `Sets the CSS <pre>max-height</pre> property.`,
+        type: 'string',
+    },
+    {
+        property: 'minHeight',
+        description: `Sets the CSS <pre>min-height</pre> property.`,
+        type: 'string',
+    },
+    {
+        property: 'minWidth',
+        description: `Sets the CSS <pre>min-width</pre> property.`,
+        type: 'string',
+    },
+    {
+        property: 'borderRadius',
+        description: `Sets the CSS <pre>border-radius</pre> property.`,
+        type: 'string',
+    },
+    {
+        property: 'display',
+        description: `Sets the CSS <pre>display</pre> property.`,
```

Comment:
For `display` it may be worth mentioning that it defaults to `inherit` if its a Text Skeleton, and `inline-block` for Non-text Skeletons